chase/frontend/app/globals.scss

```diff
@@ -10,4 +10,17 @@
 
 body {
   font-family: var(--font-family);
+}
+
+.tabular-nums {
+  font-variant-numeric: tabular-nums;
+  font-feature-settings: "tnum";
+}
+
+.custom-scrollbar {
```

Comment:
Custom scrollbars are not supported in all browsers, use with care

Reply:
It was a prime react component - I just wanted to adapt the color iof the default component and followed the docs

Reply:
Ok, just keep in mind that this might not work everywhere, should not really impact the user in any way but e.g. referring in text to the "bright red scrollbar on the right" might not be a good idea